.github/workflows/notify-amount-transferred--test.yaml

```diff
@@ -49,8 +49,4 @@ jobs:
           aqua_version: v0.10.2
       - run: github-comment exec -- yarn install --frozen-lockfile
       - run: github-comment exec -- yarn fix
-      - uses: int128/update-generated-files-action@main
-        id: update-generated-files-action
-      - name: stop this workflow when the branch is updated
-        if: steps.update-generated-files-action.outputs.updated-sha
-        run: exit 99
```

Comment:
Since v2, this action fails if there is diff. See https://github.com/int128/update-generated-files-action/pull/18.

Reply:
Oh, good updates! Thanks!